contributors/guide/contributor-cheatsheet/README-es.md

```diff
@@ -0,0 +1,393 @@
+
+Agregar nuevos commits a tu PR activará automáticamente la repetición de las pruebas.
+
+Ocasionalmente puede haber problemas con la plataforma de CI de Kubernetes. Estos pueden ocurrir
+por una variedad de razones, incluso si tu contribución pasa todas las pruebas locales. 
+Puedes activar una repetición de las pruebas con el comando `/retest`.
+
+Para obtener más información sobre la solución de problemas de pruebas específicas, consulta la [Guía de Pruebas].
+
+
+### Labels
+
+Kubernetes utiliza [labels] para categorizar y priorizar issues y Pull Requests.
+Aplicar los labels adecuados ayudará a que tu problema o PR se priorice de manera más
+efectiva.
+
+**Referencias:**
+- [Labels]
+- [Comandos de Prow][Comandos]
+
+Labels frecuentemente utilizados:
+- [`/sig <nombre de SIG>`][kind] Asigna un [SIG][SIGs] a la propiedad del issue
+  o PR.
+- [`/area <nombre de área>`][kind] Asocia el issue o las PRs a un área específica
+  [área][labels].
+- [`/kind <categoría>`][kind] [Categoriza][labels] el problema o PR.
+
+---
+
+## Trabajar Localmente
+
+Antes de proponer una solicitud de extracción (Pull Request), deberás realizar algún trabajo 
+localmente. 
+Si eres nuevo en Git, el [tutorial de Git de Atlassian] es un buen punto de partida. 
+Como alternativa, el [tutorial Git Magic] de Stanford es una buena opción multilingüe.
+
+**Referencias:**
+- [Tutorial de Git de Atlassian]
+- [tutorial Git Magic]
+- [Flujo de trabajo de GitHub]
+- [Pruebas locales]
+- [Guía del desarrollador]
+
+### Estrategia de Branch
+
+El proyecto Kubernetes utiliza un flujo de trabajo de _"Fork y Pull"_ que es estándar en GitHub. 
+En términos de Git, tu propio fork se denomina _"`origin`"_ y el repositorio Git del proyecto real se llama _"`upstream`"_. 
+Para mantener tu branch personal (`origin`) actualizada con el proyecto (`upstream`), 
+debes configurarla en tu copia de trabajo local.
+
+#### Agregar Upstream
+
+Agrega `upstream` como un repositorio remoto y configúralo para que no puedas enviar cambios a él.
+
+```
+# Reemplaza <repositorio Git upstream> con la URL del repositorio upstream
+# Ejemplo:
+#  https://github.com/kubernetes/kubernetes.git
+#  git@github.com/kubernetes/kubernetes.git
+
+git remote add upstream <repositorio Git upstream>
+git remote set-url --push upstream no_push
+```
+
+Esto se puede verificar ejecutando `git remote -v`, 
+que mostrará tus repositorios remotos configurados.
+
+#### Mantener Tu Fork Sincronizado
+
+Obtén todos los cambios de `upstream` y _"rebasa"_ (rebase) en tu branch local `master`. 
+Esto sincronizará tu repositorio local con el proyecto `upstream`. 
+Luego, envía los cambios locales a tu `remote master`.
+
+```
+git fetch upstream
+git checkout master
+git rebase upstream/master
+git push
+```
+
+Esto es lo mínimo que debes hacer antes de crear una nueva rama para trabajar en tu función o corrección.
+
+```
+git checkout -b myfeature
+```
+
+#### Squashing Commits
```

Comment:
Maybe add a blank line to separate the title from the paragraph? As the previous ones.

```suggestion
#### Squashing Commits

```